src/backend/libs/efiling-mail-it/pom.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	
+	<groupId>ca.bc.gov.open.jag</groupId>
+	<artifactId>efiling-mail-it</artifactId>
```

Comment:
I changed artifact name from jab-mail-it to efiling-mail-it to match the other client libs.